### PR TITLE
Greetings project

### PR DESCRIPTION
**Pull request by taldr27**
### In this project I:
- Created a migration file to save greetings.
- Added some Hi world in different languages inside the `seed.rb` file.
- Created a greeting model.
- Generated an API of rails to grab a random greeting.
- Created a simple React app with redux to show the random greeting.
- Added a function to catch the random greeting.
- Added minor styling.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @taldr27,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive readme file. Update your readme with the template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,117 @@
...
+Give a ⭐️ if you like this project!
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to the code reviewers and my partners who are helping all the time.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rloterh) [inline README.md:115]: - [x] MIT license link in your ReadMe is broken, kindly endeavor to fix it.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Diego, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦   🏃 

## Highlights 👍  
💠 Connection between Rails back-end and React front-end properly established in one app!
💠 Random greetings generating correctly!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (taldr27) [reply README.md:115]: Thanks for your review @rloterh, I forgot to add the `LICENSE` file, I'll add it and ask for another review👌.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi @taldr27 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * No Linter errors :white_check_mark: 
 * Connection between Rails back-end and React front-end properly established in one app! :white_check_mark: 
 * Good Job :+1: 
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x]N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**